docs/teamshares/recipes/buttons.md

```diff
@@ -0,0 +1,216 @@
+# Button Playbook
```

Comment:
It looks like this is less a general button playbook, and more a "how to convert SharedUI::Button usages to Shoelace" playbook -- super helpful, thank you! -- but naming could be clearer 🙏

Reply:
good call 🙌